src/tests/run.sh

```diff
@@ -303,35 +210,29 @@ do
 done
 
 ################################################################################
-# Runtests
+# Set environment variables affecting tests.
+# (These should be run.py arguments.)
 ################################################################################
 
 if ((disableEventLogging == 0)); then
     export COMPlus_EnableEventLog=1
 fi
 
-export COMPlus_gcServer="$serverGC"
+if ((serverGC != 0)); then
```

Comment:
This avoids setting COMPlus_gcServer in the default case (0), to avoid unnecessary "noise"